core/src/retransmit_stage.rs

```diff
@@ -177,7 +177,7 @@ fn retransmit(
     bank_forks: &RwLock<BankForks>,
     leader_schedule_cache: &LeaderScheduleCache,
     cluster_info: &ClusterInfo,
-    shreds_receiver: &Receiver<Vec<Shred>>,
+    shreds_receiver: &Receiver<Vec</*shred:*/ Vec<u8>>>,
```

Comment:
nit: `ShredPayload`, or maybe typedef the whole thing
```
pub type ShredReceiver = Receiver<Vec<ShredPayload>>;
```
and use that throughout

Reply:
I will revisit this once follow up changes are in and this code is stable, thanks
https://github.com/solana-labs/solana/pull/26230#discussion_r910431525